### ISSUE TITLE
`set` options and $mode_system are broken

### ISSUE THREAD

**Issue by The-King-of-Toasters** [swaywm/sway#2736]
In my config, I set the following style options for `dmenu`:
```
set $dmopts -i -fn 'Liberation Mono-9' -nb '#303030' -nf '#FFFFFF' -sb '#338DD6'
...
bindsym $mod+d          exec --no-startup-id dmenu_run $dmopts
```
And these are my binds for `$mode_system`
```
set $mode_system (l) lock, (e) logout, (s) suspend, (r) reboot, (S) shutdown, (R) UEFI
...
mode "$mode_system" {
	bindsym l exec $Locker, mode "default"
	bindsym e exit
	bindsym s exec --no-startup-id systemctl suspend, mode "default"
	bindsym r exec --no-startup-id systemctl reboot, mode "default"
	bindsym Shift+s exec --no-startup-id systemctl poweroff -i, mode "default"
	bindsym Shift+r exec --no-startup-id systemctl reboot --firmware-setup, mode "default"

	# back to normal: Enter or Escape
	bindsym Return mode "default"
	bindsym Escape mode "default"
	bindsym q mode "default"
}
...
bindsym $mod+Shift+e mode "$mode_system"
```
Both of these worked on dc01e884. I've reproduced this issue on two different computers.

**Comment by RedSoxFan** [comment]
Can you please try https://github.com/swaywm/sway/pull/2735?

**Comment by The-King-of-Toasters** [comment]
Confirmed not working

**Comment by SirCmpwn** [comment]
...That's not what you were asked to test

**Comment by The-King-of-Toasters** [comment]
Oops, I thought I'd been merged. Gimme a couple of hours and ill test it properly

**Comment by The-King-of-Toasters** [comment]
This is already closed, but I'm just confirming that #2735 does fix my problem. Thanks @RedSoxFan!
